### PR TITLE
Init ns doc

### PR DESCRIPTION
**Pull request by airMeng**
As title

### CODE REVIEW

File: scripts/clang-format.py
Comment (luoyu-intel) [inline scripts/clang-format.py:1]: there are some default directory parameters, could you change their relative paths accordingly?

File: scripts/__init__.py
Comment (zhenwei-intel) [inline scripts/__init__.py:1]: moving these files will cause python api convert to fail?

Comment (airMeng) [reply scripts/__init__.py:1]: reverted

File: scripts/clang-format.py
Comment (airMeng) [reply scripts/clang-format.py:1]: updated https://github.com/intel/neural-speed/blob/cc24cfbf215d9dddb139b67c6f443ded8a563548/clang-format.py#L63

**Review by luoyu-intel — APPROVED** [review]

**Pull request merged**